server/src/test/diagnostics.test.ts

```diff
@@ -1241,6 +1241,26 @@ describe("Test validation of object references", () => {
         );
         expect(diagFilterd.length).to.equal(0);
     });
+    it("Expect no diagnostics for sopcxvr without identifier", () => {
+        const text = `
+            SopcEvr: 
+            
+            Evr: TestEvr
+		`;
+        const doc = new MockDocument(text);
+        const cnfg = parseSepticSync(doc.getText());
+        const objectInfo = metaInfoProvider.getObject("MvrList");
```

Comment:
Is the test valid? MvrList here, Evr i the document

Reply:
Good catch. Object info is not used in these particular tests, so doesn't matter but should probably be fixed in a separate issue since this recurring in many tests.